test/test_error.c

```diff
@@ -251,9 +298,20 @@ int main(int argc, char **argv)
     if (0 > H5Fclose(fid)) ERROR(H5Fclose);
 
     /* Use raw file I/O to corrupt the dataset named corrupted_data */
+
```

Comment:
I guess we could simplify but just building the corrupted data once and committing it as a data file to be used in testing. Thoughts?

Reply:
It is nice to have the program around that generated the file, I don't necessarily have a problem generating in on the fly.